bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImplTest.java

```diff
@@ -83,11 +84,11 @@ public void testPage() throws ParseException {
         testJSONExport(page, getTestExporterJSONPath(TEST_BASE, PAGE));
     }
 
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     @Override
     public void testFavicons() {
         Page page = getPageUnderTest(PAGE);
-        page.getFavicons();
+        assertEquals(new HashMap<>(), page.getFavicons());
```

Comment:
I am not sure how we should test this method since it went from throwing an exception, to now using v1 functionality. The empty HashMap is being used here just to not break the build. Should I add paths for favicons to the v2 content JSON (like what is being done for v1) or something different?